src/Stratis.Bitcoin.IntegrationTests/Wallet/CustomisableAddressGapLimitBeyondBIP44.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Xunit;
+// ReSharper disable once InconsistentNaming
+
+namespace Stratis.Bitcoin.IntegrationTests.Wallet
+{
+    public partial class Customisable_address_gap_limit_beyond_BIP44
```

Comment:
These tests are not really about the customizable gap limit but rather about 1. testing the gap and 2. transactions being found when the number of addresses goes beyond the gap limit.
If you intend to add the gap limit testing tests here, then the name is fine.

Reply:
Yes good point, I was thinking of the future - but you are right, right now it isn't - I will change the name